core-web/apps/dotcms-ui/src/app/view/components/_common/dot-page-selector/dot-page-selector.component.ts

```diff
@@ -113,7 +114,12 @@ export class DotPageSelectorComponent implements ControlValueAccessor {
         if (this.searchType === 'site') {
             const site: Site = <Site>item.payload;
             this.currentHost = site;
-            this.autoComplete.completeMethod.emit({ query: `//${site.hostname}/` });
+            const event: AutoCompleteCompleteEvent = {
+                query: `//${site.hostname}/`,
+                originalEvent: {} as Event
+            };
+            // this.autoComplete.completeMethod.emit({ query: `//${site.hostname}/` });
```

Comment:
Prob clean up, just adding it here so is not forgotted.

Reply:
Fxied!

Here's a breaking change: now `(onSelect)` doesn't send the selected item directly, instead it sends an obj with 2 values: originalEvent and value as selected item.

<img width="900" alt="Screenshot 2024-08-16 at 9 50 09 AM" src="https://github.com/user-attachments/assets/8b596631-e24f-4186-b1e1-b500e74c87e8">